src/app/home/sponsors/sponsors.component.html

```diff
@@ -0,0 +1,17 @@
+<h4 class="center-align">・{{ 'sponsors.title' | translate }}・</h4>
+<div class="row">
+  <a
+    class="col s12 m6 center-align sponsor-logo"
+    href="http://www.jalasoft.com/"
+    target="_blank"
```

Comment:
Please add rel="noopener noreferrer" to this tag. Why? Because just using target="_blank" introduces a vulnerability, the opened link will have access to window.opener object and will be able to manipulate it. Also, I just realized that the footer component html has this problem, could you please update it there too?